src/display/editor/color_picker.js

```diff
@@ -81,6 +83,14 @@ class ColorPicker {
       editor?.color ||
       this.#uiManager?.highlightColors.values().next().value ||
       "#FFFF98";
+
+    ColorPicker.#l10nColor ||= Object.freeze({
```

Comment:
Why not a map ?

Reply:
It seemed nice to have a structure that cannot be accidentally modified elsewhere in the code.